lib/web/apiserver.go

```diff
@@ -341,6 +341,69 @@ type APIHandler struct {
 // ConnectionHandler defines a function for serving incoming connections.
 type ConnectionHandler func(ctx context.Context, conn net.Conn) error
 
+func (h *APIHandler) handlePreflight(w http.ResponseWriter, r *http.Request) {
+	raddr, err := utils.ParseAddr(r.Host)
+	if err != nil {
+		return
+	}
+	publicAddr := raddr.Host()
+
+	servers, err := app.Match(r.Context(), h.handler.cfg.AccessPoint, app.MatchPublicAddr(publicAddr))
+	if err != nil {
+		h.handler.log.Error("failed to match application with public addr %s", publicAddr)
```

Comment:
I suppose it wouldn't be an error error because the app itself would still function (granted, CORS wont work, but thats the same as now). I'll switch to info